### PR TITLE
BUGFIX: Fix flow 6.0 compatibility (PSR7 change)

### PR DESCRIPTION
**Pull request by tantegerda1**
Fixes: #50

Please **do not merge** yet, there are issues I am aware of.

### CODE REVIEW

**Comment by tantegerda1** [comment]
Main issue I'm having with my changes:

- backwards compatibilty

How do I achieve this change in a way that is both flow < 6.0 and flow 6.0+ compatible? Or should that be handled by composer and a composer.json that specifically targets flow 6.0+?

File: Classes/Fusion/FormImplementation.php [hunk 1]
```diff
@@ -35,12 +33,9 @@ public function evaluate()
         $formDefinition->setRenderingOption('_fusionRuntime', $this->runtime);
         $controllerContext = $this->runtime->getControllerContext();
 
-        $response = new Response($controllerContext->getResponse());
+        $actionResponse = $controllerContext->getResponse();
```
Comment (bwaidelich) [inline Classes/Fusion/FormImplementation.php]: Actually that variable is not required any longer (see below)
```suggestion
```

File: Classes/Fusion/FormImplementation.php [hunk 2]
```diff
@@ -35,12 +33,9 @@ public function evaluate()
...
         $actionRequest = $controllerContext->getRequest();
-        if (!$actionRequest instanceof ActionRequest) {
-            throw new \RuntimeException(sprintf('The Form ControllerContext requires an instance of ActionRequest, instance of "%s" given', get_class($controllerContext->getRequest())), 1499345873);
-        }
-        $formRuntime = $formDefinition->bind($actionRequest, $response);
+        $formRuntime = $formDefinition->bind($actionRequest, $actionResponse);
```
Comment (bwaidelich) [inline Classes/Fusion/FormImplementation.php]:
```suggestion
        $formRuntime = $formDefinition->bind($actionRequest, $controllerContext->getResponse(););
```

**Comment by bwaidelich** [comment]
I just relalized that this is a duplicate of #55 
I'll close this one in favor of the other one because that fixes the composer dependencies, too.

Thanks again :)

File: Classes/Fusion/FormImplementation.php (hunk 2 above)
Comment (md-fid) [reply Classes/Fusion/FormImplementation.php]: There is an additional semicolon as typo:
```suggestion
        $formRuntime = $formDefinition->bind($actionRequest, $controllerContext->getResponse());
```

Comment (bwaidelich) [reply Classes/Fusion/FormImplementation.php]: oh yes, you're right. But I closed this one in favor of #55 anyways :)

**Pull request closed without merging**